Mathlib/Analysis/CStarAlgebra/CStarMatrix.lean

```diff
@@ -383,7 +383,9 @@ def toCLMNonUnitalAlgHom [Fintype n] :
     map_zero' := by simp
     map_mul' := by
       intros
-      simp [← MulOpposite.op_mul]
+      simp only [AddHom.toFun_eq_coe, LinearMap.coe_toAddHom, LinearMap.coe_comp,
```

Comment:
Should it be `simp? says`?

Reply:
I think I'll leave it; it doesn't seem too bad